sphinxcontrib/openapi/openapi31.py

```diff
@@ -302,7 +302,7 @@ def _get_type_from_schema(schema):
             dtype = schema["type"]
         else:
             dtype = set()
-            for t in schema["anyOf"]:
+            for t in schema.get("anyOf", schema.get("allOf")):
```

Comment:
```suggestion
            for t in schema.get("anyOf", schema.get("allOf", [])):
```

Otherwise this can fail if we have e.g. `oneOf`.